src/Microsoft.TestPlatform.CrossPlatEngine/TestEngine.cs

```diff
@@ -155,6 +155,18 @@ public ITestExtensionManager GetExtensionManager()
             return this.testExtensionManager ?? (this.testExtensionManager = new TestExtensionManager());
         }
 
+        /// <summary>
+        /// Fetches the logger manager for this engine. This manager will provide logger extensibility features that this engine supports.
+        /// </summary>
+        /// <returns>ITestLoggerManager object that helps with logger extensibility.</returns>
+        public ITestLoggerManager GetLoggerManager(IRequestData requestData)
+        {
+            return new TestLoggerManager(
```

Comment:
validate the data in IRequestData is getting logged properly, even after the request is complete.